main/src/main/scala/sbt/internal/librarymanagement/UnsafeLibraryManagementCodec.scala

```diff
@@ -0,0 +1,17 @@
+package sbt.internal.librarymanagement
```

Comment:
For the sake of code organization, sbt/sbt code should be packaged under `sbt.internal`. If it belongs to `sbt.internal.librarymanagement` it should move to sbt/lm.